scripts/release/Readme.md

```diff
@@ -0,0 +1,102 @@
+- [ ] Also commit `Cargo.lock` AFTER running a Cargo command like `cargo check --workspace`
+
+## 2. Tag the [`nushell`] repo
+> **Warning**  
+> this is maybe the most critical step of the whole release process!!
+> this step, once pushed to *GitHub* will trigger the release workflows.
+
+> **Note**  
+> in the following, `nushell` will be used to pull and push to the [`nushell`] repo,
+> e.g. the `nushell` remote would be `https://github.com/nushell/nushell` or `git@github.com:nushell/nushell`
+
+- [ ] get the latest version bump commit with `git pull nushell main`
+- [ ] run `cargo build` to check if it's ok and check last features
+- [ ] tag the project with `git tag 0.xx.0`
+- [ ] :warning: push the release tag to *GitHub* `git push nushell main --tags` :warning:
+
+:point_right: check the [CI jobs](https://github.com/nushell/nushell/actions)  
+:point_right: check that there is the same number of targets compared to [last release](https://github.com/nushell/nushell/releases/latest)
+
+## 3. Publish `nu` to *crates.io*
+- [ ] check the order of dependencies with `./scripts/release/nu_deps.nu` from the `nushell` repo
+- [ ] release the Nushell crates `./scripts/release/nu_release.nu` from the `nushell` repo
+
+> **Note**  
+> if there is a new crate, you must add it to the `github:nushell:publishing` group (`cargo owner --list`)
+
+> **Note**  
+> if a step fails
+> - ask the owner to `cargo owner --add github:nushell:publishing`
+> - edit the `nu_release.nu` script to start again where it failed
+> - re-run the script
+
+## 4. Publish the release note on the website
+> **Note**  
+> the scripts have been written in such a way they can be run from anywhere
+
+- [ ] inspect the merged PRs to write changelogs with `.scripts/release/release-note/list-merged-prs nushell/nushell`
+- [ ] reorder sections by priority, what makes the most sense to the user?
+- [ ] paste the output of  `./scripts/release/release-note/list-merged-prs nushell/nushell --label breaking-change --pretty --no-author` to the "*Breaking changes*" section
+- [ ] make sure breaking changes titles are clear enough
+- [ ] paste the output of `./scripts/release/release-note/get-full-changelog` to the "*Full changelog*" section
+- [ ] mark as *ready for review* when uploading to *crates.io*
+- [ ] land when
+    - **fully uploaded** to *crates.io*
+    - **before** the *GitHub* release
+
+## 5. Publish the release on *GitHub*
+- [ ] go to the draft release on the [release page](https://github.com/nushell/nushell/releases)
+- [ ] grab the message of [last one](https://github.com/nushell/nushell/releases/latest)
+- [ ] wait for the website to publish the release (in the [actions](https://github.com/nushell/nushell.github.io/actions) tab and on the [website](https://www.nushell.sh/blog/))
+- [ ] publish the release on *GitHub*
+
+## 6. social media
+- [ ] post a status update on Discord
+- [ ] tweet about the new release using `./scripts/release/this_week_in_nu_weekly.nu`
```

Comment:
this_week_in_nu_weekly.nu is not used during the release. It's just the script used for the weekly update post.

Reply:
wow i completely misunderstood the goal of that script :scream: :laughing: 
let me correct this... :recycle:

Reply:
done in 1234f6187